java/openjdk/Portfile

```diff
@@ -420,10 +372,13 @@ by adding the following line to your shell profile:
     export JAVA_HOME=${target}/Contents/Home
 "
 
-if {[string match openjdk12* ${subport}] || [string match openjdk13* ${subport}] || [string match openjdk14* ${subport}]} {
+# Enable if a non-LTS openjdk version is not obsoleted when its subsequent version is added
+if {0} {
+if {[string match openjdk15* ${subport}]} {
     notes-append "
     Warning: Support for OpenJDK ${major} has reached end of life and there will be no more updates.
              Please consider migrating to a supported OpenJDK version.
              Currently OpenJDK 8, 11 and 15 are supported.
     "
 }
+}
```

Comment:
I disabled but retained these notes in case future openjdk ports are added without immediately obsoleting older version(s). Hopefully no one makes ports dependent on non-LTS versions so that these notes aren't needed.

Reply:
I’d be fine with obsoleting `openjdk15*` as soon as OpenJDK 15 reaches end of life and we add `openjdk16*`, so completely removing these notes would also be fine with me.

Reply:
Alright, I have removed proposed removing them. Ideally no one will make any ports dependent on openjdk15 or future non-LTS versions.